wagtailmenus/templates/menus/bootstrap4/main_menu.html

```diff
@@ -0,0 +1 @@
+{% extends 'menus/bootstrap4/main_menu_dropdown.html' %}
```

Comment:
If there's no difference between this and `main_menu_dropdown.html`, I think it might just be simpler to just rename `main_menu_dropdown.html` to `main_menu.html`.